clients/go/commands/createInstance_command.go

```diff
@@ -0,0 +1,120 @@
+package commands
+
+import (
+    "context"
+    "encoding/json"
+    "fmt"
+
+    "github.com/zeebe-io/zeebe/clients/go/pb"
+    "time"
+)
+
+const (
+    LatestVersion = -1
+    ForceLatestVersion = -2
+)
+
+type DispatchCreateInstanceCommand interface {
+    Send() (*pb.CreateWorkflowInstanceResponse, error)
+}
+
+type CreateInstanceCommandStep1 interface {
+    BPMNProcessId(string) CreateInstanceCommandStep2
+    WorkflowKey(int64) CreateInstanceCommandStep3
+}
+
+type CreateInstanceCommandStep2 interface {
+    Version(int32) CreateInstanceCommandStep3
+    LatestVersion() CreateInstanceCommandStep3
+    LatestVersionForce() CreateInstanceCommandStep3
+}
+
+type CreateInstanceCommandStep3 interface {
+    DispatchCreateInstanceCommand
+
+    // Expected to be valid JSON string
+    PayloadFromString(string) DispatchCreateInstanceCommand
+
+    // Expected to construct a valid JSON string
+    PayloadFromStringer(fmt.Stringer) DispatchCreateInstanceCommand
+
+    // Expected that object is JSON serializable
+    PayloadFromObject(interface{}) (DispatchCreateInstanceCommand, error)
+    PayloadFromMap(map[string]interface{}) (DispatchCreateInstanceCommand, error)
+}
+
+type CreateInstanceCommand struct {
+    request *pb.CreateWorkflowInstanceRequest
+    gateway pb.GatewayClient
+}
+
+func (cmd *CreateInstanceCommand) PayloadFromString(payload string) DispatchCreateInstanceCommand {
+    cmd.request.Payload = payload
+    return cmd
+}
+
+func (cmd *CreateInstanceCommand) PayloadFromStringer(payload fmt.Stringer) DispatchCreateInstanceCommand {
+    cmd.request.Payload = payload.String()
+    return cmd
+}
+
+func (cmd *CreateInstanceCommand) serializePayload(payload interface{}) (*string, error) {
+    b, err := json.Marshal(payload)
+    if err != nil {
+        return nil, err
+    }
+    jsonStringPayload := string(b)
+    return &jsonStringPayload, nil
+}
+
+func (cmd *CreateInstanceCommand) PayloadFromObject(payload interface{}) (DispatchCreateInstanceCommand, error) {
+    jsonStringPayload, err := cmd.serializePayload(payload)
+    if err != nil || jsonStringPayload == nil {
+        return nil, err
+    }
+    cmd.request.Payload = *jsonStringPayload
+    return cmd, nil
+}
+
+func (cmd *CreateInstanceCommand) PayloadFromMap(payload map[string]interface{}) (DispatchCreateInstanceCommand, error) {
+    return cmd.PayloadFromObject(payload)
+}
+
+func (cmd *CreateInstanceCommand) Version(version int32) CreateInstanceCommandStep3 {
+    cmd.request.Version = version
+    return cmd
+}
+
+func (cmd *CreateInstanceCommand) LatestVersion() CreateInstanceCommandStep3 {
+    cmd.request.Version = LatestVersion
+    return cmd
+}
+
+func (cmd *CreateInstanceCommand) LatestVersionForce() CreateInstanceCommandStep3 {
```

Comment:
This method is not supported anymore with the latest workflow deployment changes in the broker and was also removed from the java API with https://github.com/zeebe-io/zeebe/commit/edcfbb02743566cd4b7ede7643082237db247f0f#diff-1137e2ae25ebb2c456575400eb01b06cL76

I apparently forgot to delete the constant from the Java API which I will catch up on that later

Reply:
Ok, I removed it.